code/modules/mob/living/carbon/xenomorph/charge_crush.dm

```diff
@@ -327,9 +327,9 @@
 /datum/action/xeno_action/ready_charge/bull_charge
 	charge_type = CHARGE_BULL
 	speed_per_step = 0.1
-	steps_for_charge = 3
+	steps_for_charge = 5
 	max_steps_buildup = 10
-	crush_living_damage = 40 // This is multiplied by speed, which in this case will range from 0.1 to 1.
+	crush_living_damage = 30 // This is multiplied by speed, which in this case will range from 0.1 to 1.

```

Comment:
Will speed still range from 0.1 to 1 now?

Reply:
Pretty sure it will.

Reply:
This will vary between `speed_per_step * steps_for_charge` and `max_steps_buildup * steps_for_charge`
The comment wasn't accurate before and it's even less with this change.
Could you fix it?